server/app/views/api/v1/profiles/show.json.jbuilder

```diff
@@ -0,0 +1,11 @@
+
+
+#json.partial! 'profile', profile: @current_user
+
```

Comment:
```suggestion
```

This is commented-out, can it be removed?

Reply:
agree and updated